labelbox/client.py

```diff
@@ -1578,6 +1578,35 @@ def get_catalog_slice(self, slice_id) -> CatalogSlice:
         res = self.execute(query_str, {'id': slice_id})
         return Entity.CatalogSlice(self, res['getSavedQuery'])
 
+    def unarchive_feature_schema_node(self, ontology_id: str,
+                                      root_feature_schema_id: str) -> bool:
+        """
+        Returns true if the root feature schema node was successfully unarchived.
+        Only root level feature schema nodes can be unarchived.
+        Args:
+            root_feature_schema_id (str): The ID of the root level feature schema
+            ontology_id (str): The ID of the ontology
+        Returns:
+            bool
+        """
+        ontology_endpoint = self.rest_endpoint + "/ontologies/" + urllib.parse.quote(
+            ontology_id) + '/feature-schemas/' + urllib.parse.quote(
+                root_feature_schema_id) + '/unarchive'
+        response = requests.patch(
+            ontology_endpoint,
+            headers=self.headers,
+        )
+        if response.status_code == requests.codes.ok:
+            unarchived = bool(response.json()['unarchived'])
+            if (unarchived == False):
```

Comment:
You cna do:
```
if not unarchived
```

Reply:
[Updated](https://github.com/Labelbox/labelbox-python/pull/939/commits/65b814bf8d3e78793c4d83a26ea2c4cb3248fe39)